Modules/resource.c

```diff
@@ -15,6 +15,13 @@
 
 #define doubletime(TV) ((double)(TV).tv_sec + (TV).tv_usec * 0.000001)
 
+#define ADD_INT(module, value)                         \
```

Comment:
Please define this macro inside resource_exec(), around line 351. Bonus point if you undefine it at resource_exec exit, inside the body of the function. Something like:

```
resource_exec
{
#define ADD_INT ...
(...)
#undef ADD_INT
}
```

Reply:
thanks, updated.